packages/neuron-wallet/src/startup/initApp.ts

```diff
@@ -21,14 +21,24 @@ const syncConnectStatus = () => {
 }
 
 const initApp = async () => {
-  nodeService.start()
   // TODO: this function should be moved to somewhere syncing data
   syncConnectStatus()
   WalletChannel.start()
   const activeId = await NetworksController.service.activeId()
   if (!activeId) {
     await NetworksController.service.init()
   }
+  if (!nodeService.core.node.url) {
+    const id = await NetworksController.service.activeId()
```

Comment:
why not use service instead of controller

Reply:
> why not use service instead of controller

Yeah, it's indeed the service instance, which has been mounted on the controller.

Reply:
I means use service directly, not use `controller.service`

Reply:
> I means use service directly, not use `controller.service`

The service was instantiated in the Controller, you mean instantiating one more here?

Reply:
or just create one somewhere, and controller also use that one

Reply:
I think it's better to remove controller in logic level, but we can do this later

Reply:
make sense. I'm going to use DI in the future.